### DISCUSSION TITLE
the new screen is ok but

### DISCUSSION THREAD

**Discussion by theatomicboy81** [UkrainianBanderasCat/Digital-Logic-Sim2#25] [Ideas]
the screen is not very intuitive. if i where to make it i would have maybe Address  A-D are the x cord and E-G are the y cord AND EXPLAIN THIS IN THE INSTRUCTIONS. have a nice day! :)                                                                          ;)

**Comment by firecerne** [comment]
I'll look at that

**Reply by UkrainianBanderasCat** [reply]
Why? Why did you say that just now? I spent the whole day thinking about how the address works :) LOL

**Comment by AlphaCodingPilot** [comment]
I'm sorry if it wasn't clear how the screen works. I'll implement the coordinates system you suggusted. Thanks for the feedback.

**Reply by theatomicboy81** [reply]
youre welcome

**Reply by theatomicboy81** [reply]
oh and this can be used to shift text or pic (plz do 15x15 cus 4 bits by 4 bits)

**Reply by AlphaCodingPilot** [reply]
I dont know about 15x15 pixels on a screen. I think a screen should be wider than it is tall. I think 12/8 (or 3/2) is a good ratio.

**Comment by UkrainianBanderasCat** [comment]
![Screenshot from 2022-06-12 00-42-20](https://user-images.githubusercontent.com/48175036/173205864-254f288b-a888-414e-9253-2dd22d2cb23e.png)
It's seems like guy in sunglasses with long hair ;)

**Reply by theatomicboy81** [reply]
![Screenshot_7](https://user-images.githubusercontent.com/107222563/173210402-cae0bd8a-eda1-4f97-a5b5-ba43d3351119.png)
First color abstract posted pic of screen ;)
